vispy/visuals/graphs/layouts/circular.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+"""
+Circular Layout
+===============
+
+This module contains several graph layouts which rely heavily on circles.
+"""
+
+import numpy as np
+
+from ..util import straight_line_vertices
+
+
+def circular(adjacency_mat, directed=False):
+    """Places all nodes on a single circle.
+
+    Parameters
+    ----------
+    adjacency_mat : matrix
+        The graph adjacency matrix
```

Comment:
If `directed` is false, then it uses the whole matrix (for each combination of (i, j) both (i, j) as (j, i) should be non-zero).

Weights is indeed something we should implement.